event-system/tests/publishSpec.js

```diff
@@ -1,32 +1,79 @@
 import test from 'blue-tape';
+import express from 'express';
+import bodyparser from 'body-parser';
+import sinon from 'sinon';
 import request from 'supertest-as-promised';
-import {publicApi, privateApi} from '../src/index.js';
+import redis from './redis.js';
+import publish from '../src/publish.js';
 
-const before = test;
-const after = test;
+const beforeEach = (dbName) => {
+  publish.__Rewire__('redis', {createClient: redis.createClient.bind(null, dbName)});
```

Comment:
the babel rewire plugin adds that to modules. it allows me to replace the real redis connection with a fakeredis one. Since we are using then-redis, the redis.js file in the tests folder does some work to replace the redis in then-redis with fakeredis. Most of the code is copy pasted from the then-redis constructor.
Here is where I added the plugin:
https://github.com/the-ninja-turtles/turtle/pull/35/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R41
And where I enabled it for test tasks:
https://github.com/the-ninja-turtles/turtle/pull/35/files#diff-3078a3c98b0f4ac84ec62fc2817bb53fR7